lib/openid_connect.ex

```diff
@@ -515,14 +528,24 @@ defmodule OpenIDConnect do
   issuer = claims["iss"]          # Identifies the token issuer
   ```
 
+  Ignoring "aud" claim verification:
+
+  ```elixir
+  google_config = %{client_id: "aaccaecd-fd29-46e7-be3e-58a99f355157", ...}
+
+  {:ok, claims} = OpenIDConnect.verify(google_config, id_token, ignore_claims: ["aud"])
+
+  audience = claims["aud"]        # b52bc817-2ab3-4aba-bd6e-286713d542f0
+  ```
+
   ## Security Warning
 
   Always verify tokens before trusting their contents. Never use token data for
   authentication purposes without verification, as tokens could be forged or tampered with.
   """
-  @spec verify(config(), jwt :: String.t()) ::
+  @spec verify(config(), jwt :: String.t(), verify_opts()) ::
           {:ok, claims :: map()} | {:error, term()}
-  def verify(config, jwt) do
+  def verify(config, jwt, opts \\ []) do
```

Comment:
I'd expect to see a validation of the opts here


```suggestion
  def verify(config, jwt, opts \\ []) do
    opts = Keyword.validate!(opts, [ignore_claims: []])
```